non-msrv-tests/tests/ui/item.stderr

```diff
@@ -1,53 +1,51 @@
-error: no traits found to implement, use `#[derive_where(..)` to specify some
- --> tests/ui/item.rs:6:1
+error: empty `derive_where` found
+ --> tests/ui/item.rs:5:1
   |
-6 | struct NoAttribute<T>(PhantomData<T>);
-  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-error: unexpected option syntax
- --> tests/ui/item.rs:9:3
+5 | #[derive_where]
```

Comment:
Again, this seems consistent that not the correct item span is passed down.

Reply:
But this is probably correct as this is about `derive_where` being empty